### PR TITLE
docs-stacks: update create-or-modify stacks

### PR DESCRIPTION
**Pull request by mhamwala**

### CODE REVIEW

File: docs/stacks/create-or-modify.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ```
+
+3. Test your stack:
+
+    ```bash
+    cd ../templates/sample
+    appsody run
+    ```
+
+Note: If you are contributing a new template for an existing stack, just open a pull request to the in the stacks repo.
```
Comment (helenmasters) [inline docs/stacks/create-or-modify.md]: to the in the?

File: docs/stacks/create-or-modify.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Create or Modify"
+path: /docs/stacks/create-or-modify
+section: Stacks
+---
+# Creating and Modifying Stacks
+
+Use the following samples to help you get started on creating or modyifying stacks. Learn more about the [structure of a stack](stack-structure.md).
```
Comment (helenmasters) [inline docs/stacks/create-or-modify.md]: "Use the following examples to help you get started creating, or modifying, stacks."

File: docs/stacks/create-or-modify.md (hunk 1 above)
Comment (mhamwala) [reply docs/stacks/create-or-modify.md]: fixed

File: docs/stacks/create-or-modify.md (hunk 2 above)
Comment (mhamwala) [reply docs/stacks/create-or-modify.md]: addressed change

**Comment by helenmasters** [comment]
In this doc in the top of both sections (creating and modifying) we need to add (even if just one sentence) something about why you might want to create a stack and why you might need to modify a stack.

File: docs/stacks/create-or-modify.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Create or Modify"
+path: /docs/stacks/create-or-modify
+section: Stacks
+---
+# Creating and Modifying Stacks
+
+Use the following examples to help you get started creating, or modifying stacks. Learn more about the [structure of a stack](stack-structure.md).
```
Comment (helenmasters) [inline docs/stacks/create-or-modify.md]: This is where we should have the comment about why you might want to create or modify a stack (at a high level). 

We also should change the "Learn more about the [structure of a stack]..." into something more like:

"Stacks contain specific files laid out in a particular directory tree, you can learn more about these in our guide to the [Structure of a stack]. This knowledge will be helpful if you'd like to create or modify stacks."

File: docs/stacks/create-or-modify.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Note: please ensure that you have read the [contributing](../../CONTRIBUTING.md) and [code of conduct](../../CODE-OF-CONDUCT.md) guidelines before creating or modifying a stack.
```
Comment (helenmasters) [inline docs/stacks/create-or-modify.md]: This needs to be converted into more encouraging wording for users - we want to encourage users to "contribute new stacks and modify existing stacks" and to ask for help on Slack if they need help.

File: docs/stacks/create-or-modify.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+---
+title: "Create or Modify"
+path: /docs/stacks/create-or-modify
+section: Stacks
+---
+# Creating and Modifying Stacks
+
+You might want to modify an existing stack to suit your development needs, for example you might want to use a different library or runtime version from and existing stack. We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody slack]() or create a new GitHub issue to track the discussion.
```
Comment (helenmasters) [inline docs/stacks/create-or-modify.md]: from and existing stack -> from an existing stack

Comment (helenmasters) [reply docs/stacks/create-or-modify.md]: slack -> Slack

File: docs/stacks/create-or-modify.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
+---
+title: "Create or Modify"
+path: /docs/stacks/create-or-modify
+section: Stacks
+---
+# Creating and Modifying Stacks
+
+You might want to modify an existing stack to suit your development needs, for example you might want to use a different library or runtime version from an existing stack. We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody Slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
+
+We always welcome any contributions.
```
Comment (helenmasters) [inline docs/stacks/create-or-modify.md]: This and the following paragraph could be combined. Also is the following paragraph the same as we've rippled into the other docs?

File: docs/stacks/create-or-modify.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+---
+# Creating and Modifying Stacks
+
+You might want to modify an existing stack to suit your development needs, for example you might want to use a different library or runtime version from an existing stack. We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody Slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
+
+We always welcome any contributions. 
+
+If you wanted to [create](#Creating-a-stack) or [modify](#Modifying-a-stack) a stack, please review the [contributing guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE-OF-CONDUCT.md).
+
+Use the following examples to help you get started creating, or modifying stacks. Stacks contain specific files laid out in a particular directory tree, you can learn more about these in our guide of the [Structure of a stack](stack-structure.md). This knowledge will be helpful if you'd like to create or modify stacks.
```
Comment (helenmasters) [inline docs/stacks/create-or-modify.md]: guide of -> guide to

Comment (mhamwala) [reply docs/stacks/create-or-modify.md]: could this be: 
"learn more about these in our [Structure of a stack](stack-structure.md) guide.""

**Pull request merged**